### PR TITLE
updated ajax file

### PR DESCRIPTION
**Pull request by MKergosien**

### CODE REVIEW

File: front-end-notes/ajax.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## What is AJAX?
+
+#### AJAX is an acronym standing for Asynchronous JavaScript and XML. The technology helps us to load data from the server without a browser page refresh. This technology is used is popular sites such as Facebook.
+
+
```
Comment (ddbunc2) [inline front-end-notes/ajax.md]: Maybe Show an example ajax call here to refer to

Comment (rhondae1) [inline front-end-notes/ajax.md]: I would LOVE to see and example of an ajax call in code, and instructions if there's something I need to install in my project.

Comment (Dan-Hodges) [inline front-end-notes/ajax.md]: I second Deric's suggestion.

**Comment by tombler** [comment]
I'd really appreciate having a code example of a simple jQuery Ajax call or http request at the beginning of the article. I often have to look up the exact syntax and the names of the chained functions that go after (.success, .done, .then, .fail, etc.)

**Comment by sqfish** [comment]
It's probably worth mentioning that AJAX calls require JQuery. You may want to elaborate on XML HTTP request (vanilla javascript version)

**Comment by elizabrock** [comment]
+1 on including some reference examples

**Pull request merged**